Resources/Prototypes/Vanilla/Damage/modifier_sets.yml

```
@@ -2,4 +2,15 @@
   id: ChemExpert
   coefficients:
     Poison: 0.6
-    Caustic: 0.6
\ No newline at end of file
+    Caustic: 0.6
+
+- type: damageModifierSet
+  id: Trottine
+  coefficients:
+    Blunt: 0.6
+    Slash: 0.7
+    Piercing: 0.7
+    Shock: 0.7
+    Heat: 0.7
+    Cold: 0.6
+    Caustic: 0.7
```

```suggestion
    Blunt: 0.6
    Slash: 0.8
    Cold: 0.6
    Piercing: 0.8
```
я думаю так логичнее;